ui/src/attestation/contract.ts

```diff
@@ -20,7 +20,7 @@ import * as error from "ui/src/error";
 
 const addresses = {
   claims: {
-    local: "0x785e8de68df899d77ce689f863e4166849c8bfd5",
+    local: "0xff1d4d289bf0aaaf918964c57ac30481a67728ef",
```

Comment:
> Is this just a made-up address that's always deployed by the ganache daemon?

This address is determined by the local contract deployment. It depends on the account and order with which the contract deployments happen.

> It might be a bit less confusing if it was defined in `scripts/ethereum-dev-node.ts` and then imported here.

That doesn’t really work well. We don’t want to make the code in `./ui` dependent on some script. (In particular, the script requires node specific code)

> I expected this to be the source for the address from a consumer's perspective.

Can you elaborate on what you mean by this?

I know this is a bit brittle and I’d like to follow up on this in another PR
* Consolidate all the contract addresses in one file
* Document how the local addreses are determined
* Add checks to the deployment code in `scripts/ethereum-dev-node.ts` and `cypress/plugins/ethereumDevNode/index.ts` that check the addresses so that the developer is informed when they diverge.

Reply:
> That doesn’t really work well. We don’t want to make the code in ./ui dependent on some script. (In particular, the script requires node specific code)

Ah, gotcha, never mind then.

> Can you elaborate on what you mean by this?

So far all the defined contract address constants were used to connect to the contract and interact with it.
But now this constant was used to actually create a new contract, I expected it to be located somewhere closer to where the creation happens. But as you already mentioned: we don't want to import stuff from scripts. So all good.

> I know this is a bit brittle and I’d like to follow up on this in another PR

> - Consolidate all the contract addresses in one file
> - Document how the local addreses are determined
> - Add checks to the deployment code in scripts/ethereum-dev-node.ts and cypress/plugins/ethereumDevNode/index.ts that check the addresses so that the developer is informed when they diverge.

Works for me 👍